.github/workflows/e2e.install.yaml

```diff
@@ -0,0 +1,34 @@
+name: E2E - Chart Install
+
+on:
+  pull_request:
+    paths:
+      - "**"
+      - "!**.md"
+  push:
+    branches:
+      - feat/*
```

Comment:
Maybe adding `master` as well?

Reply:
We just only enable E2E testing on pull request and nightly build. If E2E testing on pull request was pass, there is no need to do the same thing on master branch.

Reply:
BTW, I'm going to remove `feat/*` branches config here.